src/dotnet-xunit/Program.cs

```diff
@@ -349,9 +349,12 @@ int RunTargetFramework(string testProject, string targetFramework, bool amendOut
                 if (runtimeFrameworkVersion == "2.0")
                     runtimeFrameworkVersion = "2.0.0";
 
-                var fxVersion = FxVersion ?? runtimeFrameworkVersion;
-                WriteLine($"Running .NET Core {fxVersion} tests for framework {targetFramework}...");
-                return RunDotNetCoreProject(outputPath, assemblyName, targetFileName, extraArgs, fxVersion, $"netcoreapp{version.Major}.0");
+                if (runtimeFrameworkVersion == "2.1")
```

Comment:
PLEASE tell me the team didn't fuck this up AGAIN. :angry:

Reply:
I don't know if they did or not. Was just being defensive.